src/standard_library/cyptographic_primitives.md

```diff
@@ -0,0 +1,103 @@
+
+# Cryptographic primitives
+
+Some cryptographic primitives are already developed and ready-to-use for any Noir project:
+
+## sha256
+
+Given an array of bytes, returns the sha256 of it:
+
+```rust
+// x = [72, 101, 108, 108, 111, 44, 32, 119, 111, 114, 108, 100, 33] (Hello, world!)
+fn main(x : [u8; 13]) {
+    constrain x == x;
```

Comment:
This constrain looks unnecessary. I see it in the examples below as well. Is there a reason for it?

Reply:
Yeah I just put it there to shut up the compiler, then copied it without removing them 🤦‍♂️